hammer/tech/__init__.py

```diff
@@ -790,6 +790,27 @@ def get_available_libraries(self) -> List[Library]:
         return list(self.tech_defined_libraries) + list(
             map(lambda el: el.store_into_library(), self.get_extra_libraries()))
 
+    # TODO refactor the now-duplicated code in  process_library_filter
```

Comment:
Is this still a TODO?

Reply:
no, I don't think so. I might need to clean it up more before upstreaming but that doesn't prevent pushing this out for the lab